frontend/src/lib/components/icons.tsx

```diff
@@ -1239,3 +1239,25 @@ export function IconCorporate(props: React.SVGProps<SVGSVGElement>): JSX.Element
         </svg>
     )
 }
+
+export function IconTuning(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+    return (
+        <svg width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+            <path
+                d="M0 14V16H6V14H0ZM0 2V4H10V2H0ZM10 18V16H18V14H10V12H8V18H10ZM4 6V8H0V10H4V12H6V6H4ZM18 10V8H8V10H18ZM12 6H14V4H18V2H14V0H12V6Z"
+                fill="currentColor"
+            />
+        </svg>
+    )
+}
+
+export function IconExport(props: React.SVGProps<SVGSVGElement>): JSX.Element {
+    return (
+        <svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
Icons should be standardized on `width="1em" height="1em"` (ideally also `viewBox="0 0 24 24"`).
Also, I wonder where they come from? Most of our icons come from Material Design now, so they've got comments like `/** Material Design Refresh icon. */`. This export one looks a lot like Material "File Download", is that correct?

Reply:
oo nice catch, these are definitely material icons. I'd just copied them over from Figma but didn't know their source. Updating these